pages/settings/index.html

```diff
@@ -36,6 +36,20 @@
 	<div class="spacer">
 	</div>
 
+	<div class="spacer"></div>
+	<div class="container padding" id="appearance-settings-container">
+		<h3>Appearance</h3>
+
+		<div class="setting-section">
+			<input type="checkbox" id="checkbox-dark-mode" />
+			<label for="checkbox-dark-mode">Enable dark mode.</label>
```

Comment:
Can you remove the period from the end of this description, to match the other settings?